src/app/pages/FrontPage/index.test.jsx

```diff
@@ -34,7 +34,10 @@ let pageData;
 beforeEach(async () => {
   fetch.mockResponse(JSON.stringify(frontPageDataPidgin));
 
-  const response = await getInitialData('some-front-page-path');
+  const response = await getInitialData({
```

Comment:
Check other occurrences of `getInitialData` to ensure an object is passed. Seems to not fail with the old string format.